pom.xml

```diff
@@ -18,46 +18,53 @@
+        <jackson.version>2.17.1</jackson.version>
         <!-- Logging -->
-        <log4j-bom.version>2.17.0</log4j-bom.version>
-        <structured-logging.version>1.9.12</structured-logging.version>
+        <log4j-bom.version>3.0.0-alpha1</log4j-bom.version>
+        <structured-logging.version>3.0.8</structured-logging.version>
         <avro.version>1.11.3</avro.version>
-        <ch-kafka.version>1.4.4</ch-kafka.version>
-        <kafka-models.version>1.0.12</kafka-models.version>
-        <ojdbc6.version>11.2.0.3</ojdbc6.version>
+        <ch-kafka.version>3.0.2</ch-kafka.version>
+        <kafka-clients.version>3.7.0</kafka-clients.version>
+        <ojdbc11.version>23.4.0.24.05</ojdbc11.version>
         <junit-jupiter.version>5.7.1</junit-jupiter.version>
         <mockito-junit-jupiter.version>3.8.0</mockito-junit-jupiter.version>
-        <commons-lang3.version>3.10</commons-lang3.version>
-        <commons-io.version>2.8.0</commons-io.version>
-        <equalsverifier.version>3.1.6</equalsverifier.version>
+        <commons-lang3.version>3.14.0</commons-lang3.version>
+        <commons-io.version>2.16.1</commons-io.version>
+        <commons-compress.version>1.26.2</commons-compress.version>
+        <equalsverifier.version>3.16.1</equalsverifier.version>
         <sqs-common-version>0.0.1</sqs-common-version>
         <imageio.version>1.4.0</imageio.version>
+        <grpc-context.version>1.64.0</grpc-context.version>
+        <nimbus-jose-jwt.version>9.40</nimbus-jose-jwt.version>
+        <snappy-java.version>1.1.10.5</snappy-java.version>
+        <guava.version>33.2.1-jre</guava.version>
+        <mockserver-client-java.version>5.5.4</mockserver-client-java.version>
 
-        <testcontainers.version>1.15.3</testcontainers.version>
+        <testcontainers.version>1.19.8</testcontainers.version>
 
         <!-- Spring -->
-        <spring-boot-dependencies.version>2.7.17</spring-boot-dependencies.version>
-        <spring-test.version>5.2.0.RELEASE</spring-test.version>
-        <spring-boot-maven-plugin.version>${spring-boot-dependencies.version}</spring-boot-maven-plugin.version>
+        <spring-boot.version>3.2.6</spring-boot.version>
+        <spring-test.version>6.1.8</spring-test.version>
 
         <!-- Maven -->
-        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-        <maven-failsafe-plugin.version>2.22.2</maven-failsafe-plugin.version>
-        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
-        <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
+        <maven-surefire-plugin.version>3.2.5</maven-surefire-plugin.version>
+        <maven-failsafe-plugin.version>3.2.5</maven-failsafe-plugin.version>
+        <maven-compiler-plugin.version>3.13.0</maven-compiler-plugin.version>
+        <jacoco-maven-plugin.version>0.8.12</jacoco-maven-plugin.version>
+        <jib-maven-plugin.version>3.2.1</jib-maven-plugin.version>
```

Comment:
Latest version of jib plugin is [3.4.3](https://mvnrepository.com/artifact/com.google.cloud.tools/jib-maven-plugin/3.4.3)

Reply:
done